### PR TITLE
Add builder-logic for EL9-based Distros

### LINKED ISSUE #642: [ENHANCEMENT] Add AWS Builder for RHEL 9 x86
With the AMIgen9 project now able to produce Amazon Machine Images for RHEL 9 x86_64 and arm_64, need to add an AWS builder for RHEL 9 x86

### PR DESCRIPTION
**Pull request by ferricoxide**
Fixes #642 

Adds builders for RHEL 9, OL 9 and CentOS 9-Stream

### CODE REVIEW

File: spel/minimal-linux.pkr.hcl [hunk 1]
```diff
@@ -1028,6 +1226,46 @@ build {
...
+      "SPEL_EXTRARPMS=${local.amigen9_extra_rpms}",
+      "SPEL_FIPSDISABLE=${var.amigen_fips_disable}",
+      "SPEL_GRUBTMOUT=${var.amigen_grub_timeout}",
+      "SPEL_USEDEFAULTREPOS=${var.amigen_use_default_repos}",
+    ]
+    execute_command = "{{ .Vars }} sudo -E /bin/sh '{{ .Path }}'"
+    only = [
+      "amazon-ebs.minimal-centos-9stream-hvm",
+      "amazon-ebs.minimal-ol-9-hvm",
+      "amazon-ebs.minimal-rhel-8-hvm",
```
Comment (lorengordon) [inline spel/minimal-linux.pkr.hcl:1262]:
```suggestion
      "amazon-ebs.minimal-rhel-9-hvm",
```

Comment (ferricoxide) [reply spel/minimal-linux.pkr.hcl:1262]: Goddamn copy-paystah miss!

**Comment by lorengordon** [comment]
go codebuild go

**Review by lorengordon — COMMENTED** [review]
Maybe could eliminate some redundancy in vars/functions between amigen9-build.sh and builder-prep-9.sh, but not going to sweat that. Changeset looks fine overall. Just want to get a working codebuild execution to post, then will approve.

**Review by lorengordon — APPROVED** [review]

**Pull request merged**